checkpoints/nine.patch

```diff
@@ -0,0 +1,61 @@
+diff --git a/backend/controllers/follows.js b/backend/controllers/follows.js
+index 89020f8..4fbfce7 100644
+--- a/backend/controllers/follows.js
++++ b/backend/controllers/follows.js
+@@ -90,7 +90,7 @@ module.exports.getFollowing = async (req, res, next) => {
+     });
+
+     if (!user) {
+-      throwException(`User ${req.params.handle} does not exist!`, 404);
++      // Task 2: Error Handling: if user does not exist
+     }
+
+     let following = await user.getFollowing({
+@@ -100,19 +100,14 @@ module.exports.getFollowing = async (req, res, next) => {
+         "handle",
+         "bio",
+         [
+-          Sequelize.literal(
+-            `EXISTS(SELECT * FROM follows WHERE followerId = ${user.id} AND followingId = User.id)`
+-          ),
+-          "isFollowedByUser"
++          // Task 3: Write query to fetch the users and check if current user is being followed or not
```

Comment:
I would suggest keeping only the query a TODO. The remaining `Sequelize.literal(`EXISTS(` part can stay as it is